modules.js

```diff
@@ -1,6 +1,5 @@
 var ui = {};
-var target = document.getElementById('target');
-
+var target = document.getElementById("target");
```

Comment:
I changed nothing about this file, it's just that my prettier change single quotes for double quotes so no need to pay attention to changes in this file.